grpc/src/main/java/com/replica/util/RouterConverters.java

```diff
@@ -224,6 +224,23 @@ public static Request toGHPtRequest(PtRouteRequest request) {
         ghPtRequest.setBetaTransfers(request.getBetaTransfers());
         ghPtRequest.setMaxVisitedNodes(request.getMaxVisitedNodes() == 0 ? DEFAULT_MAX_VISITED_NODES : request.getMaxVisitedNodes());
 
+        // Set access and egress leg modes if they've been explicitly provided.
+        // Note: even if modes other than walk are requested, Graphhopper will return these legs
+        // as Trip.WalkLeg objects
+        // Note: GraphHopper currently only accepts profiles with standard "base" names for
+        // access/egress modes. Therefore, "mode" and "profile" are somewhat interchangeable here
+        String accessMode = request.getAccessMode().equals("") ? "foot" : request.getAccessMode();
+        String egressMode = request.getEgressMode().equals("") ? "foot" : request.getEgressMode();
+        ghPtRequest.setAccessProfile(accessMode);
+        ghPtRequest.setEgressProfile(egressMode);
+
+        // If beta access/egress time hasn't been explicitly set, assume caller is using older protos
```

Comment:
This keeps backwards compatibility with clients using the older protobuf definitions, where the older `beta_walk_time` param is being used